### PR TITLE
Display build metadata on start-up

### PR DESCRIPTION
**Pull request by jpds**
Added BUILD_DATE to build metadata and log version on application start-up.

### CODE REVIEW

File: version/version.go [hunk 1]
```diff
@@ -14,7 +14,14 @@
...
 package version
 
-import "github.com/coreos/go-semver/semver"
+import (
+	"fmt"
+	"os"
+	"runtime"
+
+	"github.com/coreos/go-semver/semver"
+	"github.com/Sirupsen/logrus"
```
Comment (tboerger) [inline version/version.go]: :-1: drone is using zerolog

Comment (tboerger) [reply version/version.go]: Is it imported there? Nope.

Comment (tboerger) [reply version/version.go]: Damn, on the server it's still used

Comment (jpds) [reply version/version.go]: Yeah, it quite literally is: https://github.com/drone/drone/blob/master/cmd/drone-server/server.go#L46

Comment (jpds) [reply version/version.go]: Also, speaking from experience, JSON logs are a _terrible_ idea.

Comment (tboerger) [reply version/version.go]: From our experience they are working pretty well.

**Comment by bradrydzewski** [comment]
zerolog supports json output and pretty printed output, so best of both worlds :)

**Comment by jpds** [comment]
Switched `version.go` to `zerolog`.

**Comment by tphoney** [comment]
Hi @jpds its been a while since this has been touched, apologies for this. Due to the number of conflicts, i am going to close this PR. If you still feel this is necessary, please rebase, and we can re-review this work. 

Thank you for your patience.

**Pull request closed without merging**